samples/widgets/sales-dashboard-widgets/RevenueByProductWithUserPreferences/webpack.config.js

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+const path = require('path');
+const webpack = require('webpack');
+var CopyWebpackPlugin = require('copy-webpack-plugin');
+
+module.exports = {
+    context: path.resolve(__dirname, './src'),
+    entry: {
+        index: './RevenueByProductWithUserPreferences.jsx'
+    },
+    output: {
+        path: path.resolve(__dirname, './dist/RevenueByProductWithUserPreferences/'),
+        filename: 'RevenueByProductWithUserPreferences.js'
+    },
+    module: {
+        loaders: [
+            {
+                test: /\.html$/,
+                use: [{loader: 'html-loader'}]
+            },
+            {
+                test: /\.js$/,
+                exclude: /node_modules/,
+                use: [
+                    {
+                        loader: 'babel-loader',
+                        query: {
+                            presets: ['es2015', 'react']
+                        }
+                    }
+                ]
+            },
+            {
+                test: /\.(png|jpg|svg|cur|gif|eot|svg|ttf|woff|woff2)$/,
+                use: ['url-loader']
+            },
+            {
+                test: /\.jsx?$/,
+                exclude: /(node_modules)/,
+                loader: 'babel-loader',
+                query: {
+                    presets: ['es2015', 'react']
+                }
+            },
+            {
+                test: /\.css$/,
+                use: ['style-loader', 'css-loader']
+            },
+            {
+                test: /\.scss$/,
+                use: [{loader: 'style-loader'}, {loader: 'css-loader'}, {loader: 'sass-loader'}]
+            }
+
```

Comment:
remove unnecessary new line

Reply:
fixed with 4a7ed67e33bafe933e1d67a67d0920516a350894

Reply:
done in 4cc2bf516aa03beece86f9278ee88e1b22c97aba